ansible_collection/hpe/nimble/plugins/module_utils/hpe_nimble.py

```diff
@@ -306,3 +306,24 @@ def get_initiator_group_id(client_obj, ig_name):
         if resp is None:
             raise Exception(f"Invalid value for initiator group {ig_name}")
         return resp.attrs.get("id")
+
+
+def is_array_version_above_or_equal(array_obj_client, arr_version_to_check):
+    if arr_version_to_check is None:
+        return False
+    resp = array_obj_client.get()
+    if resp is None:
+        return False
+    array_version = resp.attrs.get("version")
+    if array_version is None:
+        return False
+
+    temp = array_version.split('-')
+    array_version = temp[0]
+    arr_version = array_version.split('.')
+    version_to_check = arr_version_to_check.split('.')
+    if arr_version[0] > version_to_check[0]:
+        return True
+    elif arr_version[0] >= version_to_check[0] and arr_version[1] >= version_to_check[1]:
+        return True
```

Comment:
Is it enough to only check major.minor?  We should probably check major.minor.maintenance.

Reply:
have modified the code to check for major.minor.mainten..